src/pages/recite-passage/recite-passage.ts

```diff
@@ -345,6 +343,34 @@ export class RecitePassagePage {
     this.fetchPassage();
   }
 
+  onAudioToggle = () => {
+    if (!this.passageAudio) {
+      const passageUrl = `http://www.esvapi.org/v2/rest/passageQuery?key=TEST&output-format=mp3&passage=${ this.reference.replace(' ', '.')}`
```

Comment:
Should not deploy the test key there!!! This is why we code review.

@johnraftery - is there a way of putting the ESV key into an `env` file?

Reply:
That would be nice, but from a quick Google, it seems not simple.

Reply:
Can't it just be something like `data.json` with `{ 'esvkey': '########' }` in it? Then read that into the global context?

Reply:
How do you read this into the global context? I didn't find that in the documentation.

Reply:
What do you make of this. Simple implementation, will do for now.
https://github.com/johnraftery/BibleTrainer/pull/31/commits/de8cc87a77141a3e28c7a275f9de953885ba2d3b